### PR TITLE
bugs: LT-2775, LT-2732

### PR DESCRIPTION
**Pull request by iichr**
LT-2775: ensure licence expiry dates are valid
LT-2732: show incorporated on the application summary/check your answers page
+ remove UK as a selectable country on OIELs

### CODE REVIEW

File: applications/forms/countries.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def countries_form(request, application_id):
             Filter(),
             Checkboxes(
                 name="countries[]",
-                options=get_countries(request, True),
+                options=get_countries(request, True, ["GB"]),
```
Comment (iichr) [inline applications/forms/countries.py:17]: exclude GB as a possible destination on OIELs

File: applications/helpers/check_your_answers.py [hunk 2]
```diff
@@ -227,6 +227,7 @@ def _convert_goods(goods, is_exhibition=False):
         if is_exhibition:
             goods_dict["Product type"] = good["other_item_type"] if good["other_item_type"] else good["item_type"]
         else:
+            goods_dict["Incorporated"] = friendly_boolean(good["is_good_incorporated"])
```
Comment (iichr) [inline applications/helpers/check_your_answers.py:230]: LT-2732 - show incorporated on the application summary/check your answers page

File: core/builtins/custom_tags.py [hunk 3]
```diff
@@ -91,17 +92,12 @@ def str_date_only(value):
...
-    month = start_date.month
-
-    for _ in range(months):
-        month += 1
-        if month == 13:
-            year += 1
-            month = 1
-
-    new_date = datetime.date(year=year, month=month, day=start_date.day)
+    new_date = start_date + relativedelta(months=+months)
```
Comment (iichr) [inline core/builtins/custom_tags.py:100]: LT-2775 - ensure licence expiry dates are valid - clarified intended behaviour re end of month and business months up to the boundary with BAs

File: applications/forms/countries.py (hunk 1 above)
Comment (jfaracik) [reply applications/forms/countries.py:17]: Should this be conditional on whether or not its a transhipment?

**Comment by iichr** [comment]
We no longer have OITranshipmentLs, `countries_form` is used only on OIELs

**Pull request merged**